roles/pywps/templates/pywps.cfg.j2

```diff
@@ -38,3 +39,6 @@ format = %(asctime)s] [%(levelname)s] %(message)s
 {% if item.extra_config is defined %}
 {{ item.extra_config }}
 {% endif %}
+
+[jobqueue]
+pause = 5
```

Comment:
Not make jobqueue pause configurable? Example like the rest `{{ item.jobqueue_pause | default('5') }}`.  It might be 5 sec is optimal, you know it better than me so it's fine to keep it hardcoded as well.

Reply:
done.